docs/changelog.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+* **18 Feb 2019:**
+  * Added a `build.sh` script to each worker so that it is possible to build a single worker with the script.
+  * Added a `download_dependencies.sh` script to `SpatialOS/scripts/` for downloading schema compiler, standard library schemas and SDKs. 
+  * Added a `generate-schema-descriptor.sh` script to `SpatialOS/scripts/` for generating a schema descriptor from the schemas. 
+  * Added a `utils.sh` script to `SpatialOS/scripts` for storing paths to workers, tools, schemas.
+  * Added a `build-project.sh` script to `SpatialOS/scripts/`. It calls other scripts to build the project.
+  * Removed `build.sh` and `clean.sh`.
```

Comment:
Should we add "...from `SpatialOS/scripts/`", if that's where they were? (Because in the first bullet point you mention adding `build.sh, so we should probably be super clear where we've added it to/removed it from.)

Reply:
thanks for spotting that! changing...

Reply:
done